File: src/main/java/client/database/data/CharacterGuildData.java
package client.database.data;

public class CharacterGuildData {
   private int id;

   private int guildId;

   private int guildRank;

   private String name;

   private int allianceRank;

   private int level;

   private int job;

   public CharacterGuildData(int id, int guildId, int guildRank, String name, int allianceRank, int level, int job) {
      this.id = id;
      this.guildId = guildId;
      this.guildRank = guildRank;
      this.name = name;
      this.allianceRank = allianceRank;
      this.level = level;
      this.job = job;
   }

   public int getId() {
      return id;
   }

   public int getGuildId() {
      return guildId;
   }

   public int getGuildRank() {
      return guildRank;
   }

   public String getName() {
      return name;
   }

   public int getAllianceRank() {
      return allianceRank;
   }

   public int getLevel() {
      return level;
   }

   public int getJob() {
      return job;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/CharacterGuildData.scala
package client.database.data

class CharacterGuildData(private var _id: Int, private var _guildId: Int, private var _guildRank: Int,
                         private var _name: String, private var _allianceRank: Int, private var _level: Int,
                         private var _job: Int) {
  def id: Int = _id

  def guildId: Int = _guildId

  def guildRank: Int = _guildRank

  def name: String = _name

  def allianceRank: Int = _allianceRank

  def level: Int = _level

  def job: Int = _job
}
